index.js

```diff
@@ -1,8 +1,8 @@
-const babel = require("babel-core").transform
-const assign = require("object-assign")
+const babel = require('babel-core').transform
+const assign = require('object-assign')
 
 module.exports = function () {
-  return this.filter("babel", (data, options) => {
-    return assign({ ext: ".js"}, babel(data.toString(), options))
+  return this.filter('babel', (data, options) => {
+    return assign({ ext: '.js'}, babel(data.toString(), options))
```

Comment:
Petpeeve: Nothing against single quotes, but I don't see this change really necessary. Do you mind reverting to double quotes?